recipes/sawfish/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set name = "sawfish" %}
+{% set version = "0.12.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PacificBiosciences/sawfish/releases/download/v{{ version }}/sawfish-v{{ version }}-x86_64-unknown-linux-gnu.tar.gz
```

Comment:
What is the reason to use a prebuilt binary ?

Reply:
We've found fully static binaries to be the most reliable way to distribute our tools.